libraries/botbuilder-ai/lib/luisRecognizer.js

```diff
@@ -19,36 +18,170 @@ class LuisRecognizer extends botbuilder_1.IntentRecognizer {
         else {
             this.options = Object.assign({}, appId);
         }
+        // We need to set verbose to true in order to get all intents
+        if (this.options.options) {
+            this.options.options.verbose = true;
```

Comment:
Where do the explicit options override this?

Reply:
They can't, I'm forcing verbose=true so that we'd always get all intents, and then the user can use recognizer::getTopIntent() to filter to the top intent if needed. If you want I can give back that control to the developer and they get top/multiple intents depending on whether they set verbose, do you prefer that?

Reply:
But LUIS options include a verbose flag which means you are then ignoring it.  I think we should leave control to options and the question is whether the default should be all or only the top intent.  Depends on the expected use case--we should ask Steve/CSE guys.

Reply:
Sounds reasonable, will do.